src/components/Map/Map.stories.js

```diff
@@ -18,6 +21,9 @@ export default {
   },
   parameters: {
     chromatic: {disableSnapshot: true},
```

Comment:
```suggestion
```
so, things appear to work properly in chromatic with the tile loading, let's give it at try for these stories too?